game/player.py

```diff
@@ -1,102 +1,12 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-import sys
-
-from game.card import Card, get_appearance
-from game.action import Action
-
-import importlib
-
 
 class Player:
```

Comment:
Is Player still used anywhere? Can we delete?

Reply:
I decided to keep the concept of "player" because the game needs to keep track of whose turn it is

Reply:
In my brain, "hand" is just a collection of cards, and a "hand" can't have a "turn".